--- a/etc/banner
+++ b/etc/banner
@@ -13,6 +13,6 @@
     MMMMMM       MMMMN     M           MMMMMMMMM      MMMM    MMMM
      MMMM          M                    MMMMMMM        M       M
        M
  ---------------------------------------------------------------
-   For those about to rock... OpenWrt 19.07-SNAPSHOT, r0-
+   For those about to rock... OpenWrt 19.07-SNAPSHOT
  ---------------------------------------------------------------
--- a/etc/openwrt_release
+++ b/etc/openwrt_release
@@ -1,7 +1,7 @@
 DISTRIB_ID='OpenWrt'
 DISTRIB_RELEASE='19.07-SNAPSHOT'
-DISTRIB_REVISION='r0-'
+DISTRIB_REVISION=''
 DISTRIB_TARGET='ipq807x/generic'
 DISTRIB_ARCH='aarch64_cortex-a53_neon-vfpv4'
-DISTRIB_DESCRIPTION='OpenWrt 19.07-SNAPSHOT r0-'
+DISTRIB_DESCRIPTION='OpenWrt 19.07-SNAPSHOT'
 DISTRIB_TAINTS='no-all busybox override'
--- a/usr/lib/os-release
+++ b/usr/lib/os-release
@@ -6,13 +6,13 @@
 VERSION_ID="19.07-snapshot"
 HOME_URL="https://openwrt.org/"
 BUG_URL="https://bugs.openwrt.org/"
 SUPPORT_URL="https://forum.openwrt.org/"
-BUILD_ID="r0-"
+BUILD_ID=""
 OPENWRT_BOARD="ipq807x/generic"
 OPENWRT_ARCH="aarch64_cortex-a53_neon-vfpv4"
 OPENWRT_TAINTS="no-all busybox override"
 OPENWRT_DEVICE_MANUFACTURER="OpenWrt"
 OPENWRT_DEVICE_MANUFACTURER_URL="https://openwrt.org/"
 OPENWRT_DEVICE_PRODUCT="Generic"
 OPENWRT_DEVICE_REVISION="v0"
-OPENWRT_RELEASE="OpenWrt 19.07-SNAPSHOT r0-"
+OPENWRT_RELEASE="OpenWrt 19.07-SNAPSHOT"
--- a/etc/openwrt_version
+++ b/etc/openwrt_version
@@ -1,1 +1,0 @@
-r0-
===================================================================
--- a/etc/opkg/distfeeds.conf
+++ b/etc/opkg/distfeeds.conf
@@ -1,4 +1,4 @@
-src/gz openwrt_base https://downloads.openwrt.org/releases/19.07-SNAPSHOT/packages/aarch64_cortex-a53/base
-src/gz openwrt_luci https://downloads.openwrt.org/releases/19.07-SNAPSHOT/packages/aarch64_cortex-a53/luci
-src/gz openwrt_packages https://downloads.openwrt.org/releases/19.07-SNAPSHOT/packages/aarch64_cortex-a53/packages
-src/gz openwrt_routing https://downloads.openwrt.org/releases/19.07-SNAPSHOT/packages/aarch64_cortex-a53/routing
+src/gz openwrt_base https://downloads.openwrt.org/releases/19.07.9/packages/aarch64_cortex-a53/base
+src/gz openwrt_luci https://downloads.openwrt.org/releases/19.07.9/packages/aarch64_cortex-a53/luci
+src/gz openwrt_packages https://downloads.openwrt.org/releases/19.07.9/packages/aarch64_cortex-a53/packages
+src/gz openwrt_routing https://downloads.openwrt.org/releases/19.07.9/packages/aarch64_cortex-a53/routing

