===================================================================
--- a/lib/wifi/qcawificfg80211.sh
+++ b/lib/wifi/qcawificfg80211.sh
@@ -2978,32 +2978,32 @@
 			max_power=30
 		fi
 		if [ "$bd_country_code" = "EU" ]; then
 			if [ "$bdmode" = "24G" ]; then
-				max_power=14
+				max_power=17
 			else
 				if [ "$channel" -ge 100 ]; then
-					max_power=23
+					max_power=26
 				else
-					max_power=16
+					max_power=19
 				fi
 			fi
 			if [ $ifname = "wl2" ]; then
-				max_power=13
+				max_power=16
 			fi
 		fi
 		#miwifi: reduce 3db for Brazil band1
 		if [ "$nv_country_code" = "BR" ]; then
 			if [ "$bdmode" = "5G" -a "$channel" -le 48 ]; then
-				max_power=13
+				max_power=16
 			fi
 		fi
 		#miwifi: we use band1 txpower for wifi ap mode
 		netmode=`uci get xiaoqiang.common.NETMODE`
 		if [ -n "$netmode" -a "$netmode" = "wifiapmode" -a "$bdmode" = "5G" ]; then
-			max_power=16
+			max_power=19
 			if [ "$nv_country_code" = "BR" ]; then
-				max_power=13
+				max_power=16
 			fi
 		fi
 		config_get txpwr "$device" txpwr
 		if [ "$txpwr" = "mid" ]; then
--- a/etc/init.d/dropbear
+++ b/etc/init.d/dropbear
@@ -130,10 +130,9 @@
 start_service()
 {
 	# 稳定版不能打开ssh服务
 	flg_ssh=`nvram get ssh_en`
-	channel=`/sbin/uci get /usr/share/xiaoqiang/xiaoqiang_version.version.CHANNEL`
-	if [ "$flg_ssh" != "1" -o "$channel" = "release" ]; then
+	if [ "$flg_ssh" != "1" ]; then
 		return 0
 	fi
 
 	[ -s /etc/dropbear/dropbear_rsa_host_key ] || keygen
--- a/etc/shadow
+++ b/etc/shadow
@@ -1,5 +1,5 @@
-root:$1$MULfKgY6$rdJYoUcznNYSPtoW5M5I3/:18191:0:99999:7:::
+root:$1$qtLLI4cm$c0v3yxzYPI46s28rbAYG//:18191:0:99999:7:::
 daemon:*:0:0:99999:7:::
 ftp:*:0:0:99999:7:::
 network:*:0:0:99999:7:::
 nobody:*:0:0:99999:7:::
