--- a/usr/lib/lua/luci/model/cbi/firewall/zones.lua
+++ b/usr/lib/lua/luci/model/cbi/firewall/zones.lua
@@ -9,9 +9,8 @@
 e.anonymous=true
 e.addremove=false
 e:option(Flag,"syn_flood",translate("Enable SYN-flood protection"))
 o=e:option(Flag,"drop_invalid",translate("Drop invalid packets"))
-e:option(Flag,"fullcone",translate("Enable FullCone NAT"))
 a={
 e:option(ListValue,"input",translate("Input")),
 e:option(ListValue,"output",translate("Output")),
 e:option(ListValue,"forward",translate("Forward"))
===================================================================
--- a/etc/opkg/distfeeds.conf
+++ b/etc/opkg/distfeeds.conf
@@ -1,4 +1,4 @@
-src/gz openwrt_base https://downloads.openwrt.org/releases/19.07-SNAPSHOT/packages/aarch64_cortex-a53/base
-src/gz openwrt_luci https://downloads.openwrt.org/releases/19.07-SNAPSHOT/packages/aarch64_cortex-a53/luci
-src/gz openwrt_packages https://downloads.openwrt.org/releases/19.07-SNAPSHOT/packages/aarch64_cortex-a53/packages
-src/gz openwrt_routing https://downloads.openwrt.org/releases/19.07-SNAPSHOT/packages/aarch64_cortex-a53/routing
+src/gz openwrt_base https://downloads.openwrt.org/releases/19.07.9/packages/aarch64_cortex-a53/base
+src/gz openwrt_luci https://downloads.openwrt.org/releases/19.07.9/packages/aarch64_cortex-a53/luci
+src/gz openwrt_packages https://downloads.openwrt.org/releases/19.07.9/packages/aarch64_cortex-a53/packages
+src/gz openwrt_routing https://downloads.openwrt.org/releases/19.07.9/packages/aarch64_cortex-a53/routing
--- a/etc/index.htm
+++ b/etc/index.htm
@@ -702,10 +702,10 @@
 				);
 
 			if (e = document.getElementById('memfree'))
 				e.innerHTML = progressbar(
-					(info.memory.free / 1048576) + " <%:MB%>",
-					(info.memory.total / 1048576) + " <%:MB%>"
+					Math.floor(info.memory.free / 1048576) + " <%:MB%>",
+					Math.floor(info.memory.total / 1048576) + " <%:MB%>"
 				);
 
 			if (e = document.getElementById('membuff'))
 				e.innerHTML = progressbar(
