--- a/lib/xqwhc/xqwhc_sync.sh
+++ b/lib/xqwhc/xqwhc_sync.sh
@@ -94,10 +94,17 @@
 
     ieee80211w_5g="`uci -q get wireless.@wifi-iface[0].ieee80211w`"
     [ -z "$ieee802211w_5g" ] && ieee802211w_5g=""
 
-    support160="`uci -q get misc.wireless.support_160m`"
-    [ -z "$support160" ] && support160=0
+    support160="`uci -q get wireless.wifi0.support160`"
+    [ -z "$support160" ] && {
+        local HARDWARE=`/sbin/uci get /usr/share/xiaoqiang/xiaoqiang_version.version.HARDWARE`
+        if [ "$HARDWARE" = "R3600" -o "$HARDWARE" = "RA69" ]; then
+            support160=1
+        else
+            support160=0
+        fi
+    }
 
     [ "$USE_ENCODE" -gt 0 ] || {
     # support special string escape
         ssid_2g="$(str_escape "$ssid_2g")"
