--- a/lib/wifi/qcawificfg80211.sh
+++ b/lib/wifi/qcawificfg80211.sh
@@ -2968,22 +2968,8 @@
 			iwpriv "$ifname" 11ngvhtintop 1
 		else
 			max_power=30
 		fi
-		if [ "$bd_country_code" = "EU" ]; then
-			if [ "$bdmode" = "24G" ]; then
-				max_power=14
-			else
-				if [ "$channel" -ge 100 ]; then
-				   max_power=23
-				else
-				   max_power=16
-				fi
-			fi
-		    if [ $ifname = "wl2" ]; then
-        		max_power=13
-    		fi		
-		fi
 		config_get txpwr "$device" txpwr
 		if [ "$txpwr" = "mid" ]; then
 			txpower=`expr $max_power - 1`
 		elif [ "$txpwr" = "min" ]; then
@@ -3830,9 +3816,9 @@
 
 EOF
 	if [ $devidx = 0 ]; then
 		cat <<EOF
-	option channel_block_list '52,56,60,64,100,104,108,112,116,120,124,128,132,136,140,144'
+	option channel_block_list '52,56,60,64'
 EOF
 	fi
 	devidx=$(($devidx + 1))
 	done
--- a/etc/config/server_mapping
+++ b/etc/config/server_mapping
@@ -1,44 +1,44 @@
 config location dftlocation
         option location 'RU'
 
 config server 'server_CN'
-	option API api.miwifi.com
-	option S s.miwifi.com
-	option APP app.miwifi.com
-	option STUN stun.miwifi.com
-	option BROKER broker.miwifi.com
+	option API localhost
+	option S localhost
+	option APP localhost
+	option STUN localhost
+	option BROKER localhost
 
 config server 'server_IN'
-	option API in.api.miwifi.com
-	option S s.miwifi.com
-	option APP app.miwifi.com
-	option STUN in.stun.miwifi.com
-	option BROKER in.broker.miwifi.com
+	option API localhost
+	option S localhost
+	option APP localhost
+	option STUN localhost
+	option BROKER localhost
 
 config server 'server_EU'
-	option API eu.api.miwifi.com
-	option S s.miwifi.com
-	option APP app.miwifi.com
-	option STUN eu.stun.miwifi.com
-	option BROKER eu.broker.miwifi.com
+	option API localhost
+	option S localhost
+	option APP localhost
+	option STUN localhost
+	option BROKER localhost
 
 config server 'server_RU'
-	option API ru.api.miwifi.com
-	option S s.miwifi.com
-	option APP app.miwifi.com
-	option STUN ru.stun.miwifi.com
-	option BROKER ru.broker.miwifi.com
+	option API localhost
+	option S localhost
+	option APP localhost
+	option STUN localhost
+	option BROKER localhost
 
 config server 'server_SG'
-	option API sg.api.miwifi.com
-	option S s.miwifi.com
-	option APP app.miwifi.com
-	option STUN sg.stun.miwifi.com
-	option BROKER sg.broker.miwifi.com
+	option API localhost
+	option S localhost
+	option APP localhost
+	option STUN localhost
+	option BROKER localhost
 
 config server 'server_US'
-	option API us.api.miwifi.com
-	option S s.miwifi.com
-	option APP app.miwifi.com
-	option STUN us.stun.miwifi.com
-	option BROKER us.broker.miwifi.com
+	option API localhost
+	option S localhost
+	option APP localhost
+	option STUN localhost
+	option BROKER localhost
