--- a/usr/lib/lua/luci/model/cbi/firewall/zones.lua
+++ b/usr/lib/lua/luci/model/cbi/firewall/zones.lua
@@ -9,9 +9,8 @@
 e.anonymous=true
 e.addremove=false
 e:option(Flag,"syn_flood",translate("Enable SYN-flood protection"))
 o=e:option(Flag,"drop_invalid",translate("Drop invalid packets"))
-e:option(Flag,"fullcone",translate("Enable FullCone NAT"))
 a={
 e:option(ListValue,"input",translate("Input")),
 e:option(ListValue,"output",translate("Output")),
 e:option(ListValue,"forward",translate("Forward"))
===================================================================
--- a/etc/opkg/distfeeds.conf
+++ b/etc/opkg/distfeeds.conf
@@ -1,4 +1,4 @@
-src/gz openwrt_base https://downloads.openwrt.org/releases/19.07-SNAPSHOT/packages/aarch64_cortex-a53/base
-src/gz openwrt_luci https://downloads.openwrt.org/releases/19.07-SNAPSHOT/packages/aarch64_cortex-a53/luci
-src/gz openwrt_packages https://downloads.openwrt.org/releases/19.07-SNAPSHOT/packages/aarch64_cortex-a53/packages
-src/gz openwrt_routing https://downloads.openwrt.org/releases/19.07-SNAPSHOT/packages/aarch64_cortex-a53/routing
+src/gz openwrt_base https://downloads.openwrt.org/releases/19.07.9/packages/aarch64_cortex-a53/base
+src/gz openwrt_luci https://downloads.openwrt.org/releases/19.07.9/packages/aarch64_cortex-a53/luci
+src/gz openwrt_packages https://downloads.openwrt.org/releases/19.07.9/packages/aarch64_cortex-a53/packages
+src/gz openwrt_routing https://downloads.openwrt.org/releases/19.07.9/packages/aarch64_cortex-a53/routing
===================================================================
--- a/usr/lib/lua/luci/view/admin_status/index.htm
+++ b/usr/lib/lua/luci/view/admin_status/index.htm
@@ -57,9 +57,9 @@
 		local user_info = luci.sys.exec("cat /proc/net/arp | grep 'br-lan' | grep '0x2' | wc -l")
 		
 		local cpu_usage = (luci.sys.exec("expr 100 - $(top -n 1 | grep 'CPU:' | awk -F '%' '{print$4}' | awk -F ' ' '{print$2}')") or "6") .. "%"
 
-		local cpu_hz = luci.sys.exec  ("echo -e `expr $(cat /sys/devices/system/cpu/cpufreq/policy0/scaling_cur_freq) / 1000`")
+		local cpu_hz = luci.sys.exec("echo -e `expr $(cat /sys/devices/system/cpu/cpufreq/policy0/scaling_cur_freq) / 1000`")
 	
 		local eth_info = ""
 		if not has_switch then
 			eth_info = luci.sys.exec("/sbin/ethinfo")
