===================================================================
--- firstFile
+++ SecondFile
@@ -2978,32 +2978,32 @@
 			max_power=30
 		fi
 		if [ "$bd_country_code" = "EU" ]; then
 			if [ "$bdmode" = "24G" ]; then
-				max_power=14
+				max_power=17
 			else
 				if [ "$channel" -ge 100 ]; then
-					max_power=23
+					max_power=26
 				else
-					max_power=16
+					max_power=19
 				fi
 			fi
 			if [ $ifname = "wl2" ]; then
-				max_power=13
+				max_power=16
 			fi
 		fi
 		#miwifi: reduce 3db for Brazil band1
 		if [ "$nv_country_code" = "BR" ]; then
 			if [ "$bdmode" = "5G" -a "$channel" -le 48 ]; then
-				max_power=13
+				max_power=16
 			fi
 		fi
 		#miwifi: we use band1 txpower for wifi ap mode
 		netmode=`uci get xiaoqiang.common.NETMODE`
 		if [ -n "$netmode" -a "$netmode" = "wifiapmode" -a "$bdmode" = "5G" ]; then
-			max_power=16
+			max_power=19
 			if [ "$nv_country_code" = "BR" ]; then
-				max_power=13
+				max_power=16
 			fi
 		fi
 		config_get txpwr "$device" txpwr
 		if [ "$txpwr" = "mid" ]; then
