--- a/lib/wifi/qcawificfg80211.sh
+++ b/lib/wifi/qcawificfg80211.sh
@@ -244,7 +244,7 @@
 		config_get mode "$vif" mode
 		case "$mode" in
 			ap|wrap|ap_monitor|ap_smart_monitor|ap_lp_iot)
-				iwpriv "$phy" setCountryID 156
+				iwpriv "$phy" setCountryID 250
 				return 0;
 			;;
 		*) ;;
@@ -3672,7 +3672,6 @@
 	reload=0
 	hw_mode_detect=0
 	avoid_load=0
-	country_code="SG"
 	prefer_hw_mode_id="$(grep hw_mode_id \
 			/ini/internal/global_i.ini | awk -F '=' '{print $2}')"
