--- a/lib/wifi/qcawificfg80211.sh
+++ b/lib/wifi/qcawificfg80211.sh
@@ -244,7 +244,7 @@
 		config_get mode "$vif" mode
 		case "$mode" in
 			ap|wrap|ap_monitor|ap_smart_monitor|ap_lp_iot)
-				iwpriv "$phy" setCountryID 156
+				iwpriv "$phy" setCountryID 250
 				return 0;
 			;;
 		*) ;;
@@ -3837,7 +3837,7 @@
 	option macaddr	$(cat /sys/class/net/${dev}/address)
 	option hwmode	11${mode_11}
 	option htmode	'${htmode}'
-	option country	'$country_code'
+	option country	'FR'
 	option disabled '$disable'
 	option txbf '3'
 	option ax '1'
@@ -3861,6 +3861,9 @@
 	option mode	'$mode'
 	option ssid	'$ssid'
 	option encryption none
+	option ieee80211k '1'
+	option ieee80211v '1'
+	option bss_transition '1'
 	option wpsdevicename 'XiaoMiRouter'

 EOF
