--- a/usr/bin/uci2dat
+++ b/usr/bin/uci2dat
@@ -30,7 +30,7 @@
     ["EU"] = {
         ["region"]      = "1",
         ["aregion"]     = "1",
-        ["SKUenable"]   = "1"
+        ["SKUenable"]   = "0"
     },
     ["UK"] = {
         ["region"]      = "1",
@@ -1112,7 +1112,7 @@
             devscfg.CountryRegionABand = sku_test["aregion"]
             devscfg.SKUenable = sku_test["SKUenable"]

             if BD_CountryCode == "EU" then
-                devscfg.SkuTableIdx = 1
+                devscfg.SkuTableIdx = 0
             elseif BD_CountryCode == "US" then
                 devscfg.SkuTableIdx = 2
